.github/workflows/building-bundle.yml

```diff
@@ -0,0 +1,40 @@
+name: Building bundle
+
+on: [ push, pull_request ]
+
+jobs:
+  build-all:
+    name: 'Building'
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ windows-latest, ubuntu-latest, macos-latest ]
+        node-version: [ 16.x ]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+
+    - name: Install dependencies
+      run: |
+        npm clean-install
```

Comment:
This is not required, `npX` command simply uses global node cache. Extension also works without any node_modules dependencies.

Reply:
I.e. `npx vsce` will run `vsce` ***without*** installation, that's the `npx` command feature